toolkit/tools/specreader/specreader.go

```diff
@@ -606,3 +626,12 @@ func filterOutDynamicDependencies(pkgVers []*pkgjson.PackageVer) (filteredPkgVer
 
 	return
 }
+
+// convertToToolchainRpmPath updates the .rpm path to point the the toolchain directory rather than the normal out/RPMs
+// directory
+func convertToToolchainRpmPath(currentRpmPath, arch string, toolchainDir string) (toolchainPath string) {
```

Comment:
```suggestion
// convertToToolchainRpmPath updates the RPM path to point the the toolchain directory rather than the normal out/RPMs
// directory
func convertToToolchainRpmPath(currentRpmPath, arch, toolchainDir string) (toolchainPath string) {
```

Reply:
done